### DISCUSSION TITLE
How does the new jaeger-opentelemetry-collector handle span data?

### DISCUSSION THREAD

**Discussion by z7658329** [jaegertracing/jaeger#5175] [Q&A] [answered]
1. how does the new jaeger-opentelemetry-collector handle span data? seems dose not need to deploy jaeger-agent？what's the protocal now？
2. in new  jaeger-opentelemetry-collector ,will limit 65K?

**Comment by yurishkuro** [comment, marked as answer]
The protocol is HTTP or gRPC. It's the same architecture as OpenTelemetry collector.

https://github.com/jaegertracing/jaeger/issues/4843
